src/Workspaces/Core/Portable/SymbolId/SymbolKey.TupleTypeSymbolKey.cs

```diff
@@ -47,12 +48,12 @@ internal override bool Equals(TupleTypeSymbolKey other, ComparisonOptions option
                 var comparer = SymbolKeyComparer.GetComparer(options);
 
                 return SequenceEquals(other._types, _types, comparer)
-                       && ((other._names == null && _names == null)
-                           || SequenceEquals(other._names, _names, CaseInsensitiveComparison.Comparer));
+                       && SequenceEquals(other._names, _names, CaseInsensitiveComparison.Comparer);
```

Comment:
Is CaseInsensitive appropriate here?  What do we do elsewhere in symbol key resolution when we're looking at names?